### PR TITLE
about page completed

### PR DESCRIPTION
**Pull request by JayaSamuthraDevi**
modifications done in  php file only.

### CODE REVIEW

File: _partials/_about.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
+ <!-- About Start -->
+ <div class="container-xxl py-5">
+     <div class="container">
+         <div class="row g-5">
+             <div class="col-lg-6 wow fadeIn" data-wow-delay="0.1s">
+                 <div class="d-flex flex-column">
+                     <img class="img-fluid rounded w-75 align-self-end" src="img/about-1.jpeg" alt="" />
+                     <img class="img-fluid rounded w-50 bg-white pt-3 pe-3" src="img/about-1.jpeg" alt="" style="margin-top: -25%" />
```
Comment (jothiswaranoh) [inline _partials/_about.php]: change space indentation

File: _partials/_foot.php [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+<script src="lib/counterup/counterup.min.js"></script>
+<script src="lib/owlcarousel/owl.carousel.min.js"></script>
+<script src="lib/tempusdominus/js/moment.min.js"></script>
+<script src="lib/tempusdominus/js/moment-timezone.min.js"></script>
+<script src="lib/tempusdominus/js/tempusdominus-bootstrap-4.min.js"></script>
+
+<!-- Template Javascript -->
+<script src="js/main.js"></script>
+</body>
+
```
Comment (jothiswaranoh) [inline _partials/_foot.php:19]: why extra line

File: _partials/_footer.php [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+                <div class="col-md-6 text-center text-md-end">
+                    <!--/*** This template is free as long as you keep the footer author’s credit link/attribution link/backlink. If you'd like to use the template without the footer author’s credit link/attribution link/backlink, you can purchase the Credit Removal License from "https://htmlcodex.com/credit-removal". Thank you for your support. ***/-->
+                    <!-- Designed By <a class="border-bottom" href="https://htmlcodex.com">HTML Codex</a> -->
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+<!-- Footer End -->
+
```
Comment (jothiswaranoh) [inline _partials/_footer.php:58]: extra line?

File: _partials/_head.php [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  <link href="lib/owlcarousel/assets/owl.carousel.min.css" rel="stylesheet">
+  <link href="lib/tempusdominus/css/tempusdominus-bootstrap-4.min.css" rel="stylesheet" />
+
+  <!-- Customized Bootstrap Stylesheet -->
+  <link href="css/bootstrap.min.css" rel="stylesheet">
+
+  <!-- Template Stylesheet -->
+  <link href="css/style.css" rel="stylesheet">
+</head>
+
```
Comment (jothiswaranoh) [inline _partials/_head.php:34]: extra line

File: _partials/_header.php [hunk 5]
```diff
@@ -0,0 +1,112 @@
+<?php load_template("head"); ?>
+
```
Comment (jothiswaranoh) [inline _partials/_header.php:2]: extra line

File: _partials/_products.php [hunk 6]
```diff
@@ -0,0 +1,61 @@
+   <!-- Products Start -->
```
Comment (jothiswaranoh) [inline _partials/_products.php:1]: spacing

File: about.html [hunk 7]
```diff
@@ -41,7 +41,6 @@
     </div>
```
Comment (jothiswaranoh) [inline about.html]: why about.html we use php right?

**Review by jothiswaranoh — CHANGES_REQUESTED** [review]
whole files change space 2 indetatation in title add ticket number extra [js-1] like that

File: about.html (hunk 7 above)
Comment (JayaSamuthraDevi) [reply about.html]: i left that for reference just for now.

**Pull request merged**